ikenna-portfolio/portfolio/src/main/java/com/google/sps/servlets/ProjectsServlet.java

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.*;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns a random quote. */
+@WebServlet("/projects")
+public final class ProjectsServlet extends HttpServlet {
+
+  private Project project;
```

Comment:
Why do you want to keep track of the state of project with a class variable.

Reply:
This project object keeps track of the info stored in the projects container. I keep an instance so I can convert it into a json object in doGet and update it in doPost.

Reply:
I assume you only want to track info for the same user instead of all users, right? Otherwise, one user's operation will affect all other users. If so, you can use http session to maintain state for the same user.